dev/ColorPicker/ColorPicker.cpp

```diff
@@ -37,7 +37,7 @@ void ColorPicker::OnApplyTemplate()
 
     m_alphaSlider.set(GetTemplateChildT<winrt::ColorPickerSlider>(L"AlphaSlider", thisAsControlProtected));
     m_alphaSliderGradientBrush.set(GetTemplateChildT<winrt::LinearGradientBrush>(L"AlphaSliderGradientBrush", thisAsControlProtected));
-    m_alphaSliderBackgroundRectangle.set(GetTemplateChildT<winrt::Rectangle>(L"AlphaSliderBackgroundRectangle", thisAsControlProtected));
+    m_alphaSliderBackgroundRectangle.set(GetTemplateChildT<winrt::Rectangle>(L"AlphaSliderGradientBackgroundRectangle", thisAsControlProtected));
```

Comment:
did we rename a template part ? If someone has a previous template, this wont work anymore ?

Reply:
Yes, but those changes were not necessary so I reverted them since we shouldn't break customers without a good reason.